### PR TITLE
simplify received transfer folder steps

### LINKED ISSUE #34025: fix confusing steps regarding received transfer folder
get rid of calls of `And using received transfer folder of "user1" as dav path`
and %last_transfer_path%
make extra steps like `And as "user1" folder "/local_storage" should not exist in the last received transfer folder`

### PR DESCRIPTION
**Pull request by individual-it**
## Description
make the steps less confusing

## Related Issue

- Fixes #34025

## Motivation and Context
simplify your code

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/TransferOwnershipContext.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use GuzzleHttp\Message\ResponseInterface;
+use TestHelpers\HttpRequestHelper;
+use TestHelpers\WebDavHelper;
+
+require_once 'bootstrap.php';
+
+/**
+ * Steps that relate to transfering ownership
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TransferOwnershipContext.php]: s/transfering/transferring/

File: tests/acceptance/features/bootstrap/TransferOwnershipContext.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+ */
+class TransferOwnershipContext implements Context {
+	/**
+	 *
+	 * @var FeatureContext
+	 */
+	private $featureContext;
+
+	/**
+	 * @Then /^the downloaded content when downloading file "([^"]*)" for user "([^"]*)" with range "([^"]*)" from last received transfer folder should be "([^"]*)"$/
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TransferOwnershipContext.php]: s/from last/from the last/

File: tests/acceptance/features/bootstrap/TransferOwnershipContext.php [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+	 *
+	 * @param string $user
+	 * @param string $entry
+	 * @param string $path
+	 *
+	 * @return array
+	 * @throws \Exception
+	 */
+	public function asFileOrFolderShouldNotExist($user, $entry, $path) {
+		//even the entry in the folder should not exist
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TransferOwnershipContext.php]: s/even the entry/only the entry/

**Review by phil-davis — APPROVED** [review]

**Comment by individual-it** [comment]
backport in #34028

**Pull request merged**